tests/system_tests/conftest.py

```diff
@@ -23,6 +25,59 @@ class ConsoleFormatter:
     END = "\033[0m"
 
 
+@pytest.fixture(scope="session")
+def local_wandb_backend() -> LocalWandbBackendAddress:
+    """Fixture that starts up or connects to the local-testcontainer.
+
+    This does not patch WANDB_BASE_URL! Use `use_local_wandb_backend` instead.
+    """
+    return connect_to_local_wandb_backend(name="wandb-local-testcontainer")
+
+
+@pytest.fixture(scope="session")
+def local_wandb_backend_importers() -> LocalWandbBackendAddress:
+    """Fixture that starts up or connects to a second local-testcontainer.
+
+    This is used by importer tests, to move data between two backends.
+    """
+    return connect_to_local_wandb_backend(name="wandb-local-testcontainer-importers")
+
+
+@pytest.fixture(scope="session")
+def use_local_wandb_backend(
+    local_wandb_backend: LocalWandbBackendAddress,
+) -> Generator[None, None, None]:
+    """Fixture that patches WANDB_BASE_URL to point to the local container.
+
+    We use the `pytest.MonkeyPatch` context manager instead of the `monkeypatch` fixture,
+    as `monkeypatch` is strictly function-scoped and we need this to be session-scoped.
+    """
+    with pytest.MonkeyPatch.context() as session_monkeypatch:
+        session_monkeypatch.setenv("WANDB_BASE_URL", local_wandb_backend.base_url)
+        yield
```

Comment:
Interesting! The main reason I created `use_local_wandb_backend` originally was to monkeypatch the environment, which I thought could only be done in a function-scoped fixture.

I'm pretty sure we don't have any code that depends on `local_wandb_backend` _not_ patching `WANDB_BASE_URL`, so I would be happy with merging the two fixtures. The string `use_local_wandb_backend` appears 30 times in the repository and would be easy to replace (though I did like the self-explanatory phrasing of "`_ = use_local_wandb_backend`" as opposed to "`_ = local_wandb_backend`").

Can keep as-is in this PR!